PolicyAdministrationPoint/src/test/java/it/cnr/iit/ucs/pap/PolicyAdministrationPointTest.java

```diff
@@ -0,0 +1,92 @@
+/*******************************************************************************
+ * Copyright 2018 IIT-CNR
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy
+ * of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ ******************************************************************************/
+package it.cnr.iit.ucs.pap;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+
+import java.io.File;
+import java.util.Map;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import it.cnr.iit.ucs.properties.components.PapProperties;
+import it.cnr.iit.utility.FileUtility;
+
+public class PolicyAdministrationPointTest {
+
+    private static final String POLICY_FILE_LOCATION = "../res/xmls/";
+    private static final String POLICY_ID = "policy2Attributes.xml";
```

Comment:
let's change the content of all the policies to reflect the filename. in this way we can test any policy file contained in the xmls path